app/editor/src/features/content/morning-report/MorningReportFilter.tsx

```diff
@@ -38,6 +32,13 @@ export const MorningReportFilter: React.FC<IMorningReportFilterProps> = () => {
         window.location.search,
       ),
     );
+    storeFilterAdvanced(
```

Comment:
It looks like we're doing this twice, is there a reason for this (see AdvancedMorningReport)?